cmd/quickstart.go

```diff
@@ -116,23 +123,30 @@ func quickstartExec(ctx context.Context, flags QuickstartFlags) error {
 		outDir = flags.OutDir
 	}
 	var targetType string
+	var sdkClassName string
+	// Assume just one language possible during quickstart
 	for _, target := range quickstartObj.WorkflowFile.Targets {
 		targetType = target.Target
+		for _, config := range quickstartObj.LanguageConfigs {
```

Comment:
Why are we doing this. We shouldn't be needing to do this

Reply:
I just want the SDKClassName for the directory name, let me know if there is an easier way